Feature/data hub 10 18 (#1185)
SoftwareAG/c8y-docs: content/datahub/integrating-datahub-with-other-products-bundle/integration-powerbi.md

The Microsoft Power BI datasets should use the DirectQuery mode, which prevents replicating and caching the data from the data lake.
{{< /c8y-admon-info >}}

> In contrast to versions prior to 10.18, it is not required anymore to deploy a Microsoft Power BI gateway. A native connector from Power BI Web to Dremio is available now.

Reviewer: ```suggestion
In contrast to versions prior to 10.18, it is no longer required to deploy a Microsoft Power BI gateway. A native connector from Power BI Web to Dremio is available now.
```
